packages/gasket-plugin-happyfeet/README.md

```diff
@@ -50,15 +50,52 @@ module.exports = {
 
 [Happy Feet](https://github.com/asilvas/happy-feet#usage) for gasket has no default configuration.
 
+## Actions
 
-## Usage
+### getHappyFeet
 
-You can call gasket.happyFeet directly to manually change the health status for any reason
-by setting `gasket.happyFeet.status = 'unhappy'` in any context that you have the gasket
-object available to your function. 
+This action is used to configure and get a [Happy Feet](https://github.com/asilvas/happy-feet#usage) instance. 
+You can specify the various [happy-feet](https://github.com/asilvas/happy-feet#usage) config options in the gasketConfig 
+or pass the config object directly to the action. 
 
-For example, you can define middleware to run before any healthchecks to define custom
-termination logic for things not included in the default metrics monitoring.
+
+#### gasketConfig example
+
+```js
+// gasket.config.js
+module.exports = {
+   happyFeet: {
+    // https://github.com/asilvas/happy-feet#usage
+    escalationSoftLimitMin: 20, // 20s
+    escalationSoftLimitMax: 300, // 5min
+    uncaughtExceptionSoftLimit: 1,
+    uncaughtExceptionHardLimit: void 0,
+    rssSoftLimit: 0.9 * 1024 * 1024 * 1024, // ~900MB
+    rssHardLimit: 1.8 * 1024 * 1024 * 1024, // ~1.9GB
+    logOnUnhappy: true
+  }
+};
+```
+
+```js
+const happy = gasket.actions.getHappyFeet();
+```
+
+#### happyConfig example
+
+```js
+const happyConfig =  {
```

Comment:
Why have 2 different ways to add config to happy-feet? 

Wouldn't the same happy-feet instance be used throughout the app and therefore just look at the gasket config for configuration?